CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - `EncodeAeson` and `DecodeAeson` instances for `TransactionInput`, `TransactionOutput`/`TransactionOutputWithRefScript` and `PaymentPubKeyHash`([#1138](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1138))
 - `mustSendChangeToAddress` balancer constraint, allowing to explicitly set the address to send all generated change to ([#1243](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1243))
 - `mustUseUtxosAtAddress` and `mustUseUtxosAtAddresses` balancer constraints, allowing to specify addresses that should be treated like utxos sources during balancing ([#1243](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1243))
+- `Ctl.Examples.PlutusV2.ReferenceInputs` example, demonstrating the use of the `mustReferenceOutput` constraint ([#1247](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1247))
```

Comment:
Please do not add changes in examples to the changelog. The purpose of the changelog is to help users move to a new version quickly. Their applications do not depend on our internal examples, so there's no point in including it here.

Reply:
@klntsky Should we also remove any other mentions of examples from the changelog?